compiler/hermes-std/src/ConsoleReporter.test.cpp

```diff
@@ -43,3 +43,83 @@ TEST(ConsoleReporterTest, notify)
 
   ASSERT_NO_THROW(r.notify(&m));
 }
+
+TEST(ConsoleReporterTest, notify_fatal)
```

Comment:
Please add same amout of negative tests, name that end with `_NEG`.
Refer https://github.com/Samsung/ONE/pull/7937#issuecomment-962942396 why.